src/v/cluster/archival/archival_metadata_stm.cc

```diff
@@ -164,6 +171,8 @@ struct archival_metadata_stm::spillover_cmd
     static constexpr cmd_key key{9};
```

Comment:
i wonder why serde doesn't encode static fields

Reply:
presumably you could, but I suppose it's strictly wasted bits since the value wouldn't change per instance. or maybe more straightforwardly, "the same reason statics don't appear in structured bindings".

Reply:
oh yeh i think it makes sense that they do not appear in the encodings!